src/main/groovy/io/seqera/wave/service/builder/BuildRequest.groovy

```diff
@@ -313,6 +318,11 @@ class BuildRequest {
         format==SINGULARITY
     }
 
+    BuildRequest withLabels(Map<String, String> labels){
```

Comment:
with the `labels` now in the `containerConfig` attribute, I think we don't need changes to this class any more